backend/src/database/tag/tag.service.ts

```diff
@@ -2,11 +2,16 @@ import { Injectable } from '@nestjs/common';
 import { PrismaService } from '../prisma.service';
 import { Tag } from '@prisma/client';
 
+const tagLimit: number = 5
```

Comment:
Please write constants in uppercase, and add a comment that this is a constant, or add an environment variable to `.env` & `.env.example`